### PR TITLE
add scripts directory with build and run helpers

### PR DESCRIPTION
**Pull request by skpy**
These scripts help with Docker usage. Rather than remember the necessary invocations, use these to build and tag the container, as well as to stop and remove any current instance and re-create it with the newest `latest` image tag.

### CODE REVIEW

**Comment by cgloeckner** [comment]
the run-script needs some updates on avoiding logs and resource routing.
besides that, i'm fine

File: settings.example.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "shards": [ ],
+  "hosting": {
+    "domain": "example.com",
+    "port": 80,
+    "socket": "",
+    "ssl": false,
+    "reverse": false
+  },
+  "login": {
+    "type": "google",
```
Comment (cgloeckner) [inline settings.example.json]: maybe we could add `auth0` here as go-to

File: scripts/run.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+docker stop icvtt
+docker rm icvtt
+docker run -d -p 8080:8080 -v /opt/pyvtt/prod:/opt/pyvtt/prod --restart=on-failure --name icvtt pyvtt:latest
```
Comment (cgloeckner) [inline scripts/run.sh:4]: do we want to add the `--no-logs` and `--no-resource-routing` here?

File: settings.example.json (hunk 1 above)
Comment (skpy) [reply settings.example.json]: Great idea.

File: scripts/run.sh (hunk 2 above)
Comment (skpy) [reply scripts/run.sh:4]: Yes!

Comment (skpy) [reply scripts/run.sh:4]: Added `--no-logs` to the Dockerfile.

**Pull request merged**